app-i18n/uchardet/uchardet-9999.ebuild

```diff
@@ -7,10 +7,7 @@ inherit cmake-utils git-r3
 
 DESCRIPTION="An encoding detector library"
 HOMEPAGE="https://www.freedesktop.org/wiki/Software/uchardet/"
-EGIT_REPO_URI=(
-	https://anongit.freedesktop.org/git/${PN}/${PN}.git
-	git://anongit.freedesktop.org/${PN}/${PN}
-)
+EGIT_REPO_URI="https://anongit.freedesktop.org/git/${PN}/${PN}.git"
```

Comment:
@mgorny -- as the maintainer of git-r3.eclass, do you have any thoughts on how to handle git protocol priority and fallback behaviour?